packages/api/README.md

```diff
@@ -20,6 +29,8 @@ Caso queira verificar o code coverage execute:
 npm run coverage
 ```
 
+Obs.: para se conectar ao banco `sabia-testing`, o servidor utilizará os valores de usuário (`DB_USER`) e senha (`DB_PASSWORD`) definidos no arquivo `.env`.
+
 ## Framework Adonis
 
 This project was inicializado utilizando o [Framework Adonis](https://adonisjs.com/)
```

Comment:
Pode ajeitar essa linha tbm rsrs?

Reply:
Entendi não...rsrs. Tem algo que não tô percebendo aí? Algum erro, @nicholasio ?

Reply:
This project was inicializado utilizando o

Reply:
Vixe...kkkkk. Tinha percebido não!

Reply:
Pronto.